crates/router/src/connector/intuit/transformers.rs

```diff
@@ -0,0 +1,277 @@
+                expires: item.response.expires_in,
+            }),
+            ..item.data
+        })
+    }
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct IntuitPaymentsRequest {
+    amount: String,
+    currency: enums::Currency,
+    description: Option<String>,
+    context: IntuitPaymentsRequestContext,
+    card: Card,
+    capture: bool,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Card {
+    number: Secret<String, pii::CardNumber>,
+    exp_month: Secret<String>,
+    exp_year: Secret<String>,
+    cvc: Secret<String>,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct IntuitPaymentsRequestContext {
+    mobile: bool,
+    is_ecommerce: bool,
+}
+
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for IntuitPaymentsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        match item.request.payment_method_data {
+            api::PaymentMethod::Card(ref ccard) => {
+                let submit_for_settlement = matches!(
+                    item.request.capture_method,
+                    Some(enums::CaptureMethod::Automatic) | None
+                );
+                Ok(Self {
+                    amount: item.request.amount.to_string(),
+                    currency: item.request.currency,
+                    context: IntuitPaymentsRequestContext {
+                        mobile: item.request.browser_info.clone().map_or(true, |_| false),
+                        is_ecommerce: false,
+                    },
+                    card: Card {
+                        number: ccard.card_number.clone(),
+                        exp_month: ccard.card_exp_month.clone(),
+                        exp_year: ccard.card_exp_year.clone(),
+                        cvc: ccard.card_cvc.clone(),
+                    },
+                    capture: submit_for_settlement,
+                    description: item.description.clone(),
+                })
+            }
+            _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
+        }
+    }
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct IntuitPaymentsCaptureRequest {
+    amount: String,
+}
+
+impl TryFrom<&types::PaymentsCaptureRouterData> for IntuitPaymentsCaptureRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(value: &types::PaymentsCaptureRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            amount: value
+                .request
+                .amount_to_capture
+                .ok_or(errors::ConnectorError::MissingRequiredField {
+                    field_name: "amount_to_capture",
+                })?
+                .to_string(),
```

Comment:
```suggestion
            amount: value
                .request
                .get_amount_to_capture()?,
```
add get_amount_to_capture in connectors/utils.rs